.github/workflows/nightly-elixir.yml

```diff
@@ -51,13 +51,23 @@ jobs:
           mix run generate.exs elixir $SPEC_PATH $TEMPLATE_PATH $ELIXIR_OUTPUT_PATH
       - name: Commit files
         run: |
-          cd aws-beam/aws-elixir
-          git config --local user.email "aws-elixir@github.com"
-          git config --local user.name "AWS Elixir"
-          git add lib/aws/generated/*
-          if [ -n "$(git status --porcelain)" ]
+          cd aws/aws-sdk-go/
```

Comment:
`pushd` and `popd` apparently don't exist in Github workflows so I have to manually `cd` into the directory and pop back out on line 57. Not the prettiest but it'll have to do

Reply:
An alternative is to hold `pwd` in a variable and `cd` back. WDYT?